fix_includes.py

```diff
@@ -2368,15 +2369,16 @@ def main(argv):
                           ' [default]'))
   parser.add_option('--noblank_lines', action='store_false', dest='blank_lines')
 
-  parser.add_option('--comments', action='store_true', default=False,
-                    help='Put comments after the #include lines')
-  parser.add_option('--nocomments', action='store_false', dest='comments')
+  parser.add_option('--comments', default='short',
+                    help='Put comments after the #include lines'
```

Comment:
Python trick: use parentheses around multiline strings to help indent them correctly -- this is such common practice that most python editors do the right thing:
```
parser.add_option('--comments', default='short',
                    help=('Put comments after the #include lines'
                              ' options: {none, short, long} default: short'))
```